verification/curator-service/Dockerfile

```diff
@@ -42,12 +42,12 @@ ENV REACT_APP_PUBLIC_MAPBOX_TOKEN "pk.eyJ1IjoiaGVhbHRobWFwIiwiYSI6ImNrYzNjczdmcz
 ENV REACT_APP_POLICY_PUBLIC_ID "89575059"
 ENV REACT_APP_COOKIE_CONSENT_PUBLIC_ID "2070778"
 ENV REACT_APP_RECAPTCHA_SITE_KEY "6LdhjvwgAAAAALrxavR_zR58kxxap07D4ba8X-jE"
-ENV REACT_APP_DISEASE_NAME "Mpox"
-ENV REACT_APP_MAP_LINK_DEV "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_LOCAL "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_LOCAL_E2E "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_PROD "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_QA "http://map.marburg.global.health/"
+ENV VITE_APP_DISEASE_NAME "Mpox"
+ENV VITE_APP_MAP_LINK_DEV "http://dev-map.marburg.global.health/"
+ENV VITE_APP_MAP_LINK_LOCAL "http://dev-map.marburg.global.health/"
```

Comment:
Can these be left blank or made consistent? Right now disease is Mpox but URLs are marburg

Reply:
This is an issue that I and Jim found during deploying the app. In order to deploy build is run on the CuratorUI and for build env variables are set. This problem has been already solved partially for the other variables like disease name, which is queried from backend. Same solution can be employed also for map related variables. I can create an issue for that.